src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -174,6 +181,13 @@ protected String[] customCommandLineValidation() {
                     hcArgs.getDragenNameValuePairs(),
                     HaplotypeCallerArgumentCollection.DRAGEN_GATK_MODE_LONG_NAME);
         }
+        if (hcArgs.flowMode != HaplotypeCallerArgumentCollection.FlowMode.NONE) {
```

Comment:
If dragen mode and flow mode are both selected at the same time, should we throw a `UserException`?

Reply:
Done. Resolved